packages/tradershub/src/components/ActionScreen/ActionScreen.tsx

```diff
@@ -29,12 +29,7 @@ const ActionScreen = ({
     titleSize = 'md',
 }: TActionScreenProps) => {
     return (
-        <div
-            className={qtMerge([
-                'flex flex-col items-center justify-center gap-24  w-auto h-auto rounded-xs',
-                className,
-            ])}
-        >
+        <div className={clsx(['flex flex-col items-center justify-center gap-24 w-auto h-auto rounded-xs', className])}>
```

Comment:
I believe the square brackets are unnecessary. You can simply just do it like this:
```suggestion
        <div className={clsx('flex flex-col items-center justify-center gap-24 w-auto h-auto rounded-xs', className)}>
```